README.md

```diff
@@ -110,20 +129,32 @@ Create your [Iterate](https://iteratehq.com) account if you haven't already.
 
 1. Create a new survey and select "Install in your mobile app"
 2. Go to the "Preview & Publish" tab and copy your SDK API key
-3. Wrap your App in the `withIterate` higher-order component
+3. Call Iterate.Init with your apiKey, safeArea, and storage, then wrap your App in the `<SafeAreaProvider>` (if using react-native-safe-area-context) and `<IterateProvider>` components
```

Comment:
```suggestion
3. Call Iterate.Init with your apiKey, safeArea function, and storage, then wrap your App in the `<SafeAreaProvider>` (if using react-native-safe-area-context) and `<IterateProvider>` components
```

I just had to think for a minute and go back to the previous section to remember how I'd do this if not using `react-native-safe-area-context`